api/turing/cluster/models.go

```diff
@@ -62,22 +64,19 @@ type BaseService struct {
 	InitContainers []Container `json:"init_containers"`
 }
 
-func (cfg *BaseService) buildResourceReqs(
-	UserContainerCPULimitRequestFactor float64,
-	UserContainerMemoryLimitRequestFactor float64,
-) corev1.ResourceRequirements {
+func (cfg *BaseService) buildResourceReqs() corev1.ResourceRequirements {
```

Comment:
can you help add some comment on this? feel like i will forget the order/flow of in the future

Reply:
Alright just added something. Thanks!